crates/turbo-tasks-fs/src/embed/dir.rs

```diff
@@ -36,12 +36,12 @@ macro_rules! embed_directory {
 
         // make sure the types the `include_dir!` proc macro refers to are in scope
         use turbo_tasks_fs::embed::include_dir;
-        #[cfg(debug_assertions)]
+        #[cfg(all(debug_assertions, feature = "dynamic_embed_contents"))]
```

Comment:
the issue with this is that every crate that embeds anything now needs the feature, because the macro gets executed in the scope of the crate calling it